apps/data/src/lib/enrich/enrich/backend_client.py

```
@@ -202,3 +206,56 @@ def get_user_metadata(self, user_ids: List[str]) -> Dict[str, Dict[str, Any]]:
+            batch = items[i : i + max_batch_size]
+            payload = {"items": batch, "timeout": timeout}
+            
+            try:
+                result = self._make_request(self.WEBHOOK_MACRO_BATCH_PATH, payload)
+                all_results.extend(result.get("results", []))
+                batch_errors = result.get("errors", [])
+                if batch_errors:
+                    all_errors.extend(batch_errors)
+            except BackendIntegrationError:
+                raise
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)